src/mmveri.c

```diff
@@ -461,7 +454,7 @@ nmbrString *assignVar(nmbrString *bigSubstSchemeAss,
           str(g_Statement[substScheme].numReqHyp),
           " hypotheses.  The ",tmpStr,
           ".  Type \"SHOW PROOF ",g_Statement[statementNum].labelName,
-          "\" to see the proof attempt.",NULL)); */ /* Old version */
+          "\" to see the proof attempt.",NULL)); */ // Old version
```

Comment:
any reason you didn't convert this one?

Reply:
It's a block of commented out code, so it falls in the fourth category described in #139 in case someone would want to use it for experimental use. Do you prefer it to be converted anyway?

Reply:
Hm, personally I find it easier to comment out blocks of code using single line comments because there are editor tools to do it, but I'm fine with it either way.

Reply:
Ok, so from now on I won't apply rule 4 of #139 anymore. I'll also make the conversion for the previously edited files, since there were some blocks of code in there as well.